mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -6185,7 +6195,8 @@
 													"    if(len > 0){",
 													"        //Test that count is equal to number of records returned in response",
 													"        pm.test('count equals number of records in response', function() {",
-													"            pm.expect(len).eq(5);",
+													"            pm.expect(len).to.be.above(0);",
+													"            pm.expect(len).to.be.below(6);",
```

Comment:
I think expecting the result set to be less than 6 records always is not necessary. It could go over that in future probably which will make this test brittle.

Reply:
agree, will remove ` pm.expect(len).to.be.below(6);",`

Reply:
👍